web/templates/user_group_settings/user_group_membership_status.hbs

```diff
@@ -0,0 +1,12 @@
+<span>
+    {{#if is_direct_member}}
+        {{t "You are a member of this group."}}
+    {{else if (not is_member)}}
+        {{t "You are not a member of this group."}}
+    {{else}}
+        {{#tr}}
+            You are a member of this group because you are a member of a subgroup (<z-subgroup-names></z-subgroup-names>).
+            {{#*inline "z-subgroup-names"}}{{{associated_subgroup_names_string}}}{{/inline}}
```

Comment:
Not sure if there is a better approach to keep only the names bold and handle the translation to tag complete sentence together for handling RTL languages.

Reply:
This seems OK. `associated_subgroup_names_html` would be a better variable name though to be super clear about its type.

Reply:
Updated.